src/components/core/settings-header/TemplatesSettings.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+import { Popover, Button } from "antd";
+import { useTemplates } from "../../stores/settings.store";
+
+export function TemplateSettings() {
+  const setTemplate = useTemplates((state: any) => state.setTemplate);
+
+  const content = (
+    <div>
+      <Button onClick={() => setTemplate(0)}>0</Button>
```

Comment:
```suggestion
      <Button onClick={() => useTemplates.getState().setTemplate(0)}>0</Button>
```

Reply:
setTemplate is a function . so when we initialize like const setTemplate = useTemplates((state: any) => state.setTemplate);. zustand add a listner. but for function call, we can directly call in non-reactive way

Reply:
![image](https://user-images.githubusercontent.com/12962887/122976886-7d043800-d3b2-11eb-9b52-811aae2ebcdf.png)
They have recommended like this in React components in docs. Confused which is the best approach

Reply:
yes. we can do this also. we will explore this part later